CHANGELOG.md

```diff
@@ -6,6 +6,9 @@
 
 - Fix handling of corrupted records by persistent buffer (experimental) (#4475)
 
+## 💡 Enhancements 💡
+
+- Extending the contribution guide to help clarify what is acceptable defaults and recommendations.
```

Comment:
Another point to the recommendations doc: when should a changelog entry be added?

Reply:
I think what I have in mind for this:

- Behaviourally changes
- Fixes
- Breaking changes

I things like CI changes, testing updates, internal helper package changes (provided it doesn't fall into the above three), don't really need to be consumed by a user of the collector.

Did you have something similar in mind?

Reply:
Would the changes to the contributing guide fit this? We have two main audiences for this repository: 

- component developers, interested in the API changes we introduce
- end-users, interested in the config changes for the components located here (otlpexporter and receiver, for instance)

IMO, we should have entries only for changes affecting those audiences.